test-helpers/src/shotover_process.rs

```diff
@@ -29,11 +33,32 @@ impl ShotoverProcessBuilder {
         self
     }
 
+    pub fn with_observability_address(mut self, address: SocketAddr) -> Self {
```

Comment:
This could just take a port number. Saves the `.parse().unwrap()` in the method calls.

Reply:
yep, good call